interfaces/builtin/modem_manager_test.go

```diff
@@ -121,9 +121,8 @@ func (s *ModemManagerInterfaceSuite) TestConnectedPlugSnippetUsesSlotLabelOne(c
 }
 
 func (s *ModemManagerInterfaceSuite) TestConnectedPlugSnippedUsesUnconfinedLabelOnClassic(c *C) {
-	slot := &interfaces.Slot{}
 	release.OnClassic = true
-	snippet, err := s.iface.ConnectedPlugSnippet(s.plug, slot, interfaces.SecurityAppArmor)
+	snippet, err := s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecurityAppArmor)
 	c.Assert(err, IsNil)
 	c.Assert(string(snippet), testutil.Contains, "peer=(label=unconfined),")
 }
```

Comment:
Can you add one more test `TestConnectedPlugSnippedUsesUnconfinedLabelNotOnClassic` that makes sure `"peer=(label=unconfined),"` is not present when `release.OnClassic = false`?